.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ on:
 jobs:
   pod-lint:
     name: Pod Lint
-    runs-on: macOS-13
+    runs-on: macOS-14
     steps:
       - uses: ruby/setup-ruby@v1
```

Comment:
this step is failing right now due to https://github.com/ruby/setup-ruby/issues/568 which will hopefully be resolved soon